docs/designers-developers/developers/slotfills/README.md

```diff
@@ -99,14 +99,14 @@ function PostStatus( { isOpened, onTogglePanel } ) {
 
 There are currently seven available SlotFills in the `edit-post` package. Please refer to the individual items below for usage and example details:
 
-* [PluginBlockSettingsMenuItem](./plugin-block-settings-menu-item.md)
-* [PluginDocumentSettingPanel](./plugin-document-setting-panel.md)
-* [PluginMoreMenuItem](./plugin-more-menu-item.md)
-* [PluginPostPublishPanel](./plugin-post-publish-panel.md)
-* [PluginPostStatusInfo](./plugin-post-status-info.md)
-* [PluginPrePublishPanel](./plugin-pre-publish-panel.md)
-* [PluginSidebar](./plugin-sidebar.md)
-* [PluginSidebarMoreMenuItem](./plugin-sidebar-more-menu-item.md)
+* [PluginBlockSettingsMenuItem](./plugin-block-settings-menu-item)
```

Comment:
Comparing to other links in the handbook, it seems like absolute paths are needs for the handbook to correctly process them.
```suggestion
* [PluginBlockSettingsMenuItem](/docs/designers-developers/developers/slotfills/plugin-block-settings-menu-item.md)
```

Reply:
Yes, this is correct. @lukecarbis - can you update all those links?

Reply:
@gziolo @talldan Thanks for the tip! Implemented in 5652555.